superset/__init__.py

```diff
@@ -195,21 +195,21 @@ def index(self):
 
 security_manager = appbuilder.sm
 
-results_backend = app.config.get("RESULTS_BACKEND")
-results_backend_use_msgpack = app.config.get("RESULTS_BACKEND_USE_MSGPACK")
+results_backend = app.config["RESULTS_BACKEND"]
+results_backend_use_msgpack = app.config["RESULTS_BACKEND_USE_MSGPACK"]
 
 # Merge user defined feature flags with default feature flags
-_feature_flags = app.config.get("DEFAULT_FEATURE_FLAGS") or {}
-_feature_flags.update(app.config.get("FEATURE_FLAGS") or {})
+_feature_flags = app.config["DEFAULT_FEATURE_FLAGS"] or {}
```

Comment:
I don't think we need the `or {}` here, as `DEFAULT_FEATURE_FLAGS` is `Dict[str, bool]` in `config.py`.